src/runtime/rpc/rpc_engine.cpp

```diff
@@ -38,12 +38,18 @@
 #include <dsn/tool-api/async_calls.h>
 #include <dsn/cpp/serialization.h>
 #include <dsn/utility/rand.h>
+#include <dsn/utility/flags.h>
 #include <set>
 
 namespace dsn {
 
 DEFINE_TASK_CODE(LPC_RPC_TIMEOUT, TASK_PRIORITY_COMMON, THREAD_POOL_DEFAULT)
 
+DSN_DEFINE_bool("core",
```

Comment:
what does this config item used for? when to config true and when false?

Reply:
this configuration is used for unit test to disable register rpc handlers. Because in some unit tests, one same rpc handler will register twice more times, and this will produce coredump. So you can set this configuration to false to avoid this situation.